docs/source/guide/lre-3-options.md

```diff
@@ -0,0 +1,278 @@
+ circuit,
+ degree = 2,
+ fold_multiplier = 2,
+ folding_method = fold_global)[5]
+
+
+print("original circuit: ", circuit ,sep="\n")
+print("Noise scaled circuit created using local unitary folding: ", local_fold_circ ,sep="\n")
+print("Noise scaled circuit created using global unitary folding: ", global_fold_circ ,sep="\n")
+
+
+```
+
+
+## Chunking a circuit into fewer layers
+
+
+When you have a large circuit, the size of the sample matrix increases as the number of monomial terms scale polynomially. The size of the sample matrix influences sampling cost. In such a case, a circuit of 100 layers could be grouped into 4 chunks where each chunk consists of multiple collated layers. The noise scaling function {func}`.multivariate_layer_scaling`
+treats each chunk as a layer to be scaled when the parameter `num_chunks` is used. Thus, for the 100 layer circuit grouped into 4 chunks with `degree = 2` and `fold_multiplier = 2`, only 15 noise-scaled circuits are created i.e. sample matrix is reduced to dimension $15 \times 15$.
+
+
+```{caution}
+Reducing the sampling cost by chunking the circuit can affect the performance of the technique.
+```
+
+
+Suppose we want to chunk our example circuit into 2 chunks while using `degree = 2` and
+`fold_multiplier = 2`. The sample matrix defined by the monomial terms is reduced in size as chunking the circuit
+will create a new monomial basis for the extrapolating polynomial.
+
+
+$$\{1, λ_1, λ_2, λ_1^2, λ_1 λ_2, λ_2^2\}$$
+
+
+The scale factor vectors change as shown below:
+
+
+```{code-cell} ipython3
+
+scale_factors_with_chunking = get_scale_factor_vectors(
+   circuit,
+   degree=2,
+   fold_multiplier=2,
+   num_chunks=2)
+
+scale_factors_with_chunking
+```
+
+
+Thus, the total number of noise-scaled circuits is reduced by chunking the circuit into fewer layers to be folded.
+
+
+```{code-cell} ipython3
+
+print(f"Total number of noise scaled circuits with chunking: {len(scale_factors_with_chunking)}")
+
+
+print(f"Total number of noise scaled circuits without chunking: {len(scale_factors)}")
+
+
+```
+
+
+How the noise-scaled circuits are chunked differs greatly as each chunk in the circuit is now equivalent to a layer to be folded via unitary folding.
+
+
+```{code-cell} ipython3
+
+# apply chunking
+chunked_circ = multivariate_layer_scaling(
+ circuit,
+ degree = 2,
+ fold_multiplier = 2,
+ num_chunks = 2)[1]
+
+
+# skip chunking
+non_chunked_circ = multivariate_layer_scaling(
+ circuit,
+ degree = 2,
+ fold_multiplier = 2)[1]
+
+
+print("original circuit: ", circuit ,sep="\n")
+print("Noise scaled circuit created with chunking: ", chunked_circ ,sep="\n")
+print("Noise scaled circuit created without chunking: ", non_chunked_circ ,sep="\n")
+
+
+```
+
+
+This section showed how to tune the performance of the technique by using the non-default options as well as controlling
+the default options. An in-depth discussion on these is provided in [](lre-4-low-level.md)
```

Comment:
I think it's more accurate to say we showed how the vary the parameters of the technique and what they do. As for how it alters the performance is still somewhat unknown (and no claims were made about performance besides the note about chunking).